doc/development/deprecations.rst

```diff
@@ -117,6 +101,27 @@ Other deprecations
 Completed deprecation cycles
 ----------------------------
 
+* The :class:`~pennylane.BasisStatePreparation` template has been removed.
+  Instead, use :class:`~pennylane.BasisState`.
+
+  - Deprecated in v0.39
+  - Removed in v0.40
+
+* The ``'ancilla'`` argument for :func:`~pennylane.iterative_qpe` has been removed. Instead, use the ``'aux_wire'``
+  argument.
+
+  - Deprecated in v0.39
+  - Removed in v0.40
+
+* The ``qml.workflow.set_shots`` helper func has been removed. We no longer interact with the legacy device interface in our code
+  workflow behavior. Instead, shots should be specified on the tape, and the device should pull it's number of shots from the tape.
```

Comment:
This was updated in the original PR, needs to be updated here too.

Reply:
Yeah this is so weird. I remember every suggestion has been committed. Probably this is caused by some conflict merging process